### PR TITLE
feature/eslint integration and undefined variables fixed

### PR DESCRIPTION
**Pull request by AminIdeaUsher**
Integration of ESLint
Undefined variables fixed in few files

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ const dbOptions = {
 const imgTypeToExtension = {
   'image/jpeg': 'jpg',
   'image/png': 'png',
-  'image/jpeg': 'jpeg',
```
Comment (AminIdeaUsher) [inline src/constants/index.js:11]: This key was duplicate

File: src/microservices/fileUpload.service.js [hunk 2]
```diff
@@ -2,13 +2,7 @@ const uuid = require('uuid').v4;
 const multer = require('multer');
 const storage = multer.memoryStorage();
 const {getSignedUrl} = require('@aws-sdk/s3-request-presigner');
-const {
-  S3Client,
-  PutObjectCommand,
-  GetObjectCommand,
-  DeleteObjectCommand,
-  ListObjectsV2Command,
```
Comment (AminIdeaUsher) [inline src/microservices/fileUpload.service.js:10]: ListObjectsV2Command was unused

File: src/services/auth.service.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
 const {User} = require('../models');
-const httpStatus = require('http-status');
-const ApiError = require('../utils/ApiError');
 
 async function createUser(user) {
-  return await User.create(user);
```
Comment (AminIdeaUsher) [inline src/services/auth.service.js:6]: await is redundant

File: src/services/auth.service.js [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
+  return User.create(user);
 }
 
 async function getUserByFirebaseUId(id) {
-  return await User.findOne({firebaseUid: id});
```
Comment (AminIdeaUsher) [inline src/services/auth.service.js:10]: await is redundant

File: src/validations/custom.validation.js [hunk 5]
```diff
@@ -1,6 +1,5 @@
 const Joi = require('joi');
 const {default: mongoose} = require('mongoose');
-const {imageTypes} = require('../constants');
```
Comment (AminIdeaUsher) [inline src/validations/custom.validation.js:3]: unused import

**Review by dev-ideausher — APPROVED** [review]
Never Review your own PR. make commit messages self-explanatory

**Pull request merged**